vendor/k8s.io/kubernetes/staging/src/k8s.io/apiserver/pkg/server/healthz/healthz.go

```diff
@@ -66,16 +66,25 @@ func NamedCheck(name string, check func(r *http.Request) error) HealthzChecker {
 // exactly one call to InstallHandler. Calling InstallHandler more
 // than once for the same mux will result in a panic.
 func InstallHandler(mux mux, checks ...HealthzChecker) {
+	InstallPathHandler(mux, "/healthz", checks...)
```

Comment:
Do we need to change vendor code, can't we do this stuff in openshift code?

Reply:
AFAIK changing vendor code needs 'UPSTREAM: <carry>' commit in this case.

Reply:
@pravisankar I did post an upstream PR to occupancy this [kubernetes/PR63716](https://github.com/kubernetes/kubernetes/pull/63716). The commit that includes the kubernetes code is tagged UPSTREAM:63716, should I add this tag to the PR title?

Reply:
Separate UPSTREAM:63716 commit is good, no need to add upstream tag to the PR title.